packages/geolocator/tizen/src/permission_manager.h

```diff
@@ -2,11 +2,10 @@
 // Use of this source code is governed by a BSD-style license that can be
```

Comment:
I changed the PermissionManager by referencing another package.
I added `ChangePermissionResulttoLocationPermission` and `ChangePermissionStatustoLocationPermission` because we have to keep the enum order used by the geolocator.

Reply:
You can change the values of the `PermissionStatus` enum as necessary (and modify the implementation of `PermissionManager` accordingly) without having to convert between the types. Please refer to https://github.com/flutter-tizen/plugins/pull/379. The `PermissionManager` class in the permission_handler_tizen package has been copied from image_picker_tizen, but has only a single return type `PermissionStatus` because its corresponding Dart type is already defined in the plugin's platform interface.

Reply:
I changed the PermissionStatus enum of PermissionManager.